core/src/main/java/org/bitcoinj/core/PartialMerkleTree.java

```diff
@@ -154,11 +154,10 @@ public byte[] serialize() {
      * @return size of the serialized message in bytes
      */
     public int messageSize() {
-        int size = Integer.BYTES; // transactionCount
-        size += VarInt.sizeOf(hashes.size());
-        size += hashes.size() * Sha256Hash.LENGTH;
-        size += Buffers.lengthPrefixedBytesSize(matchedChildBits);
-        return size;
+        return Integer.BYTES + // transactionCount
+                VarInt.sizeOf(hashes.size()) +
+                hashes.size() * Sha256Hash.LENGTH +
+                Buffers.lengthPrefixedBytesSize(matchedChildBits);
```

Comment:
I prefer the `+` at the beginning of the line on these multiline sums. At some point we should probably standardize this and/or look at what other projects are suggesting.

Reply:
I do love methods that are a single `return` expression!

(In a more modern functional language, the `return wouldn't even be necessary, of course)